src/composites/FeaturesContainer/Overview/OverviewDropLists.tsx

```diff
@@ -0,0 +1,81 @@
+// import { Component, ReactNode } from "react";
+import { overviewData, listNode } from './index.ts';
+import * as Accordion from "@radix-ui/react-accordion";
+import { Button, Theme, Box, Text, Strong } from '@radix-ui/themes';
+import { DotFilledIcon } from '@radix-ui/react-icons';
+import "./Overview.css"
+
+export function OverviewDropList(risk:string, section:string, props:overviewData): JSX.Element {
```

Comment:
Can we turn this into a component? It's confusing it structurally looks and acts like a component but has an inconsistent signature